xml/System.Formats.Tar/PaxTarEntry.xml

```diff
@@ -65,39 +64,33 @@ The entry type of <paramref name="other" /> is not supported in the PAX format.<
       <Docs>
         <param name="entryType">The type of the entry.</param>
         <param name="entryName">A string with the path and file name of this entry.</param>
+
         <summary>Initializes a new <see cref="T:System.Formats.Tar.PaxTarEntry" /> instance with the specified entry type and entry name, and the default extended attributes.</summary>
         <remarks>
-          <format type="text/markdown"><![CDATA[
-
-## Remarks
-
-When creating an instance using the <xref:System.Formats.Tar.PaxTarEntry.%23ctor(System.Formats.Tar.TarEntryType,System.String)> constructor, only the following entry types are supported:
-
-
-- In all platforms: <xref:System.Formats.Tar.TarEntryType.Directory>, <xref:System.Formats.Tar.TarEntryType.HardLink>, <xref:System.Formats.Tar.TarEntryType.SymbolicLink>, <xref:System.Formats.Tar.TarEntryType.RegularFile>.
-- In Unix platforms only: <xref:System.Formats.Tar.TarEntryType.BlockDevice>, <xref:System.Formats.Tar.TarEntryType.CharacterDevice> and <xref:System.Formats.Tar.TarEntryType.Fifo>.
-
-Use the <xref:System.Formats.Tar.PaxTarEntry.%23ctor(System.Formats.Tar.TarEntryType,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.String}})> constructor to include additional extended attributes when creating the entry.
-
-The following entries are always found in the Extended Attributes dictionary of any PAX entry:
-
-- Modification time, under the name `mtime`, as a <xref:System.Double> number.
-- Access time, under the name `atime`, as a <xref:System.Double> number.
-- Change time, under the name `ctime`, as a <xref:System.Double> number.
-- Path, under the name `path`, as a string.
-
-The following entries are only found in the Extended Attributes dictionary of a PAX entry if certain conditions are met:
-
-- Group name, under the name `gname`, as a string, if it is larger than 32 bytes.
-- User name, under the name `uname`, as a string, if it is larger than 32 bytes.
-- File length, under the name `size`, as an <xref:System.Int32>, if the string representation of the number is larger than 12 bytes.
-
-
-          ]]></format>
+          <para>When creating an instance using the <see cref="M:System.Formats.Tar.PaxTarEntry.#ctor(System.Formats.Tar.TarEntryType,System.String)"/> constructor, only the following entry types are supported: <see cref="F:System.Formats.Tar.TarEntryType.Directory"/>, <see cref="F:System.Formats.Tar.TarEntryType.HardLink"/>, <see cref="F:System.Formats.Tar.TarEntryType.SymbolicLink"/>, <see cref="F:System.Formats.Tar.TarEntryType.RegularFile"/>, <see cref="F:System.Formats.Tar.TarEntryType.BlockDevice"/>, <see cref="F:System.Formats.Tar.TarEntryType.CharacterDevice"/>, and <see cref="F:System.Formats.Tar.TarEntryType.Fifo"/>.</para>
```

Comment:
This unfortunately is messed up too: https://review.docs.microsoft.com/en-us/dotnet/api/system.formats.tar.paxtarentry.-ctor?view=net-7.0&branch=main#system-formats-tar-paxtarentry-ctor(system-formats-tar-tarentrytype-system-string)

Reply:
Ok I'll fix it here first, then later in main. This is the branch from which I need to retrieve the intellisense and the undoc APIs report before tomorrow.